metrics_ci_integration/pubspec.yaml

```diff
@@ -0,0 +1,17 @@
+name: ci_integration
+description: A sample command-line application.
+# version: 1.0.0
+# homepage: https://www.example.com
```

Comment:
Could you please a bit more details here, description that helps to understand this tool?